src/auth/ha-authorize.ts

```diff
@@ -156,13 +192,14 @@ export class HaAuthorize extends litLocalizeLiteMixin(LitElement) {
       }
 
       if (authProviders.length === 0) {
-        alert("No auth providers returned. Unable to finish login.");
+        this._error = "No auth providers returned. Unable to finish login.";
         return;
       }
 
       this._authProviders = authProviders;
       this._authProvider = authProviders[0];
     } catch (err: any) {
+      this._error = "Unable to fetch auth providers.";
```

Comment:
Should we translate this?

Reply:
I think we should really, the translations are probaly not loaded at this point and it really should not happen anyway. As in, it is not a user error...